src/microbe_stage/MicrobeHUD.cs

```diff
@@ -779,19 +783,20 @@ private void UpdateATP(float delta)
             return;
 
         var atpAmount = 0.0f;
-        var capacity = 4.0f;
 
+        // Update to the player's current HP, unless the player is dead
         if (stage.Player != null)
         {
             var compounds = GetPlayerColonyOrPlayerStorage();
 
-            atpAmount = Mathf.Ceil(compounds.GetCompoundAmount(atp));
-            capacity = compounds.Capacity;
+            // Only show ATP changes in increments of 0.5
+            atpAmount = Mathf.Ceil(compounds.GetCompoundAmount(atp) * 2) / 2.0f;
```

Comment:
In playtesting this part doesn't look good. It seems that the bar smoothing also doesn't work that well...

Reply:
yeah, that's an unfortunate side effect of trying to render microbes with low atp without flickering. I'll see if there's anything I can do about it.

Reply:
I think I've figured out a functional, if not very elegant, solution.